bambi/plots/plotting.py

```diff
@@ -0,0 +1,277 @@
+# pylint: disable = protected-access
+# pylint: disable = too-many-function-args
+# pylint: disable = too-many-nested-blocks
+from typing import Union
+import warnings
+
+import arviz as az
+import numpy as np
+
+from arviz.plots.backends.matplotlib import create_axes_grid
+from arviz.plots.plot_utils import default_grid
+from pandas.api.types import is_categorical_dtype, is_numeric_dtype, is_string_dtype
+
+import bambi as bmb
+from bambi.utils import listify, get_aliased_name
+from bambi.plots.effects import predictions, comparisons
+from bambi.plots.plot_types import plot_numeric, plot_categoric
+from bambi.plots.utils import get_covariates
+
+
+def plot_cap(
```

Comment:
So basically all the hard work you did on the other modules allowed you to get much lighter versions of `plot_cap()` and `plot_comparisons()`... great!